chain/ethevents/handlers.go

```diff
@@ -139,6 +139,55 @@ func HandleVochainOracle(ctx context.Context, event *ethtypes.Log, e *EthereumEv
 		}
 		log.Debugf("broadcasting tx: %s", log.FormatProto(setProcessTx))
 
+		res, err := e.VochainApp.SendTX(tx)
+		if err != nil || res == nil {
+			log.Warnf("cannot broadcast tx: (%s)", err)
+			return fmt.Errorf("cannot broadcast tx: %w, res: %+v", err, res)
+		}
+		log.Infof("oracle transaction sent, hash: %s", res.Hash)
+
+	case ethereumEventList["processesCensusUpdated"]:
+		tctx, cancel := context.WithTimeout(ctx, time.Minute)
+		defer cancel()
+		setProcessTx, err := processCensusUpdatedMeta(tctx, &e.ContractsABI[0], event.Data, e.VotingHandle)
+		if err != nil {
+			return err
```

Comment:
Wrap the errors and give some information regarding where it comes from (same for the rest)

Reply:
changed on all handlers, also added more precise info